modules/protocol/resources/META-INF/smithy/smithy4s.meta.smithy

```diff
@@ -24,3 +24,16 @@ structure packedInputs {}
 @trait(selector: "structure :not([trait|error])")
 @idRef(failWhenMissing: true, selector: "union")
 string adtMember
+
+@trait(selector: """
```

Comment:
Let's add some documentation for these annotations, in the form of a triple-slash comment (like for `@adtMember`)

Reply:
Addressed in [b3365de](https://github.com/disneystreaming/smithy4s/pull/303/commits/b3365de87c6af31653ec9810054c9d128c6a0e07)